src/main/java/Duke.java

```diff
@@ -24,29 +30,28 @@ public static void printWelcomeMessage() {
 
         System.out.println("Hello from\n" + logo);
 
-        printOutputFormat(welcomeMessage);
+        printWithTemplate(welcomeMessage);
     }
 
     public static void printByeMessage() {
         String byeMessage = "Bye. Hope to see you again soon!";
 
-        printOutputFormat(byeMessage);
+        printWithTemplate(byeMessage);
     }
 
     public static void addTask(String task) {
 
         tasks[taskCount] = new Task(task);
```

Comment:
Instead of using arrays and counting the number of tasks. You could also use arraylists from java docs to access more functionalities

Reply:
That is a good suggestion! Thank you, I will try to read up on it.